src/pages/Home.js

```diff
@@ -0,0 +1,147 @@
+import React from "react";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import "../styles/sliderStyles.css";
+import slider_diary from "../assets/home/slider/slider_diary.png";
+import slider_game from "../assets/home/slider/slider_game.png";
+import slider_gymnastics from "../assets/home/slider/slider_gymnastics.png";
+import slider_dementiacenter from "../assets/home/slider/slider_dementiacenter.png";
+import slider_survey from "../assets/home/slider/slider_survey.png";
+import btn_centerMap from "../assets/home/button/btn_centerMap.png";
+import btn_survey from "../assets/home/button/btn_survey.png";
+import btn_diary from "../assets/home/button/btn_diary.png";
+import btn_gymnastics from "../assets/home/button/btn_gymnastics.png";
+import btn_rockScissorsPaper from "../assets/home/button/btn_rockScissorsPaper.png";
+import btn_colorMatch from "../assets/home/button/btn_colorMatch.png";
+
+export const HOME_PAGE_PATH = "/home";
+
+const Home = () => {
+  return (
+    <div>
+      <SimpleSlider />
+      <div className="px-5">
+        <div className="flex justify-between my-8">
+          <MainButton adverb={"간단한"} buttonName={"치매진단"} imgsrc={btn_survey} />
+          <MainButton adverb={"하루기록"} buttonName={"일기장"} imgsrc={btn_diary} />
+        </div>
+        <DementiaCenterButton />
+        <div className="text-2xl font-bold mt-14 mb-5">치매를 예방해볼까요?</div>
+        <GymnasticsButton />
+        <div className=" text-md md:text-xl my-5">
+          <span className="text-orange-500">미니게임</span>
+          <span>을 통해 두뇌를 활성화 시키세요!</span>
+        </div>
+        <GameButtonList />
```

Comment:
```suggestion
        <DementiaPreventionSection />
```

처럼 묶어도 될듯 `DementiaPreventionSection`는 다른 파일로 빼도 되고

Reply:
이부분과 위에 메인기능 부분도 MainDementiaPreventionSection으로 변경하고 따로 파일로 분리했습니다